### ISSUE TITLE
Missing username config field, cannot connect to LRS

### ISSUE THREAD

**Issue by unicitgit** [davidpesce/moodle-logstore_xapi#188]
**Description**
- Hello, I installed the latest version on Moodle 3.5 but I cant connect to LRS and send any data.
I found 2 issues
1. It seems that the username field from Moodle config page is missing
![ll](https://user-images.githubusercontent.com/25296033/42812502-8df953a4-89c6-11e8-857e-baa128c60220.PNG)
2. I get these erros when I turn on debug mode
```
Warning: Creating default object from empty value in /var/www/html/moodle/admin/tool/log/store/xapi/version.php on line 19

Notice: Undefined property: stdClass::$id in /var/www/html/moodle/admin/tool/log/store/xapi/src/transformer/handler.php on line 32
Exception: Priviliges not sufficient for this operation in /var/www/html/moodle/admin/tool/log/store/xapi/src/loader/lrs.php:59 Stack trace: #0 /var/www/html/moodle/admin/tool/log/store/xapi/src/loader/lrs.php(77): src\loader\lrs\load_transormed_events_to_lrs(Array, Array) #1 [internal function]: src\loader\lrs\{closure}(Array, Array) #2 /var/www/html/moodle/admin/tool/log/store/xapi/src/loader/lrs.php(79): array_reduce(Array, Object(Closure), Array) #3 /var/www/html/moodle/admin/tool/log/store/xapi/src/loader/handler.php(24): src\loader\lrs\load(Array, Array) #4 /var/www/html/moodle/admin/tool/log/store/xapi/src/handler.php(36): src\loader\handler(Array, Array) #5 /var/www/html/moodle/admin/tool/log/store/xapi/classes/log/store.php(114): src\handler(Array, Array) #6 /var/www/html/moodle/admin/tool/log/store/xapi/classes/log/store.php(76): logstore_xapi\log\store->process_events(Array) #7 /var/www/html/moodle/admin/tool/log/classes/helper/buffered_writer.php(99): logstore_xapi\log\store->insert_event_entries(Array) #8 /var/www/html/moodle/admin/tool/log/classes/helper/buffered_writer.php(125): logstore_xapi\log\store->flush() #9 /var/www/html/moodle/admin/tool/log/classes/log/manager.php(195): logstore_xapi\log\store->dispose() #10 /var/www/html/moodle/lib/classes/shutdown_manager.php(78): tool_log\log\manager->dispose() #11 [internal function]: core_shutdown_manager::shutdown_handler() #12 {main}
line 85 of /admin/tool/log/store/xapi/classes/log/store.php: call to debugging()
line 40 of /admin/tool/log/store/xapi/src/handler.php: call to logstore_xapi\log\store->logstore_xapi\log\{closure}()
line 114 of /admin/tool/log/store/xapi/classes/log/store.php: call to src\handler()
line 76 of /admin/tool/log/store/xapi/classes/log/store.php: call to logstore_xapi\log\store->process_events()
line 99 of /admin/tool/log/classes/helper/buffered_writer.php: call to logstore_xapi\log\store->insert_event_entries()
line 125 of /admin/tool/log/classes/helper/buffered_writer.php: call to logstore_xapi\log\store->flush()
line 195 of /admin/tool/log/classes/log/manager.php: call to logstore_xapi\log\store->dispose()
line 78 of /lib/classes/shutdown_manager.php: call to tool_log\log\manager->dispose()
line ? of unknownfile: call to core_shutdown_manager::shutdown_handler()
```

**Version**
- $plugin->version = 2018071601;

**Steps to reproduce the bug**
1. Set up LRS in Moodle config file
2. Turn Debugging mode on
3. Go to a course
4. The errors will appear on the page

**Server information**
- MariaDB
- Apache
- PHP 7.0
- 

**Client information**
- OS: Centos 7
- Browser: Chrome

**Comment by ryasmi** [comment]
Hey @unicitgit, thanks for spotting that, my apologies didn't realise I'd created a duplicate setting name. The new zip should appear on the Moodle registry in a few minutes.

**Comment by unicitgit** [comment]
Thank you 🗡

**Comment by ryasmi** [comment]
No worries, should be fixed in 3.2.2
